src/extension/openFileAtLocation.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import {RemoteExtension} from "../common/remoteExtension";
+import {ReactNativeProjectHelper} from "../common/reactNativeProjectHelper";
+import {InternalErrorCode} from "../common/error/internalErrorCode";
+import {ErrorHelper} from "../common/error/errorHelper";
+import * as path from "path";
+
+/* Usage:
+...path\openFileAtLocation.js filename:lineNumber
+...path\openFileAtLocation.js filename
+*/
+
+try {
+    const fileInfo = process.argv[2].split(":");
+    const filename: string  = fileInfo[0];
+    let lineNumber: number = 1;
+    if (fileInfo.length >= 2) {
+        lineNumber = parseInt(fileInfo[1], 10);
+    }
+
+    getReactNativeWorkspaceForFile(filename).then(projectRootPath => {
+        const remoteExtensionServerPath = RemoteExtension.atProjectRootPath(projectRootPath);
+        return remoteExtensionServerPath.openFileAtLocation(filename, lineNumber);
+    }).done();
+} catch (e) {
```

Comment:
This catch will only catch synchronous errors, since the `.done()` will throw asynchronously. You may want to add an error handler to the `.done` to report the error.

Then again, this script is executed by the packager. What happens if the script does fail, does the packager report the error anywhere?